src/main/java/io/sealights/plugins/sealightsjenkins/BeginAnalysis.java

```diff
@@ -458,7 +456,7 @@ private void copyAgentsToSlaveIfNeeded(Logger logger, CleanupManager cleanupMana
         }
     }
 
-    private String tryGetSlMvnPluginVersion(SeaLightsPluginInfo slInfo, ILogger logger, Properties additionalProps) {
+    private String tryGetSlMvnPluginVersion(SeaLightsPluginInfo slInfo, Logger logger, Properties additionalProps) {
```

Comment:
Isn't it require an import statement? cant see one

Reply:
this is the same package, so no need for import.